spec/features/projects/modules_spec.rb

```diff
@@ -30,43 +30,36 @@
 
 RSpec.describe 'Projects module administration' do
   let!(:project) do
-    create(:project,
-           enabled_module_names: [])
+    create(:project, enabled_module_names: [])
   end
 
-  let(:permissions) { %i(edit_project select_project_modules) }
+  let(:permissions) { %i(edit_project select_project_modules view view_work_packages) }
   let(:settings_page) { Pages::Projects::Settings.new(project) }
 
   current_user do
     create(:user, member_with_permissions: { project => permissions })
   end
 
   it 'allows adding and removing modules' do
+    project_work_packages_menu_link_selector = '//ul[contains(@class, "menu_root")]//span[text()="Work packages"]'
+
     settings_page.visit_tab!('modules')
 
-    expect(page)
-      .to have_unchecked_field 'Activity'
+    expect(page).to have_unchecked_field 'Activity'
+    expect(page).to have_unchecked_field 'Work packages'
+    expect(page).to have_unchecked_field 'Calendar'
+    expect(page).to have_unchecked_field 'Time and costs'
 
-    expect(page)
-      .to have_unchecked_field 'Calendar'
-
-    expect(page)
-      .to have_unchecked_field 'Time and costs'
+    expect(page).not_to have_xpath(project_work_packages_menu_link_selector, visible: false)
 
     check 'Activity'
-
     click_button 'Save'
 
     settings_page.expect_toast message: I18n.t(:notice_successful_update)
 
-    expect(page)
-      .to have_checked_field 'Activity'
-
-    expect(page)
-      .to have_unchecked_field 'Calendar'
-
-    expect(page)
-      .to have_unchecked_field 'Time and costs'
+    expect(page).to have_checked_field 'Activity'
+    expect(page).to have_unchecked_field 'Calendar'
```

Comment:
I think it is better to have same list of checked/unchecked fields in all 3 places, also probably better in the same order